test/test_output_writers.jl

```diff
@@ -268,13 +268,13 @@ topo = (Periodic, Periodic, Bounded)
 
     for arch in archs
         grid = RectilinearGrid(arch, topology=topo, size=(4, 4, 4), extent=(1, 1, 1))
+        model = NonhydrostaticModel(grid=grid)
```

Comment:
```suggestion
        model = NonhydrostaticModel(; grid)
```

Reply:
Commit. Could you please explain why your suggestion is preferred though?